src/buffer/out/textBufferCellIterator.hpp

```diff
@@ -47,6 +48,8 @@ class TextBufferCellIterator
     const OutputCellView& operator*() const noexcept;
     const OutputCellView* operator->() const noexcept;
 
+    COORD Pos() const noexcept;
+

```

Comment:
@DHowett I imagine you're not a fan of this change haha. An iterator _should not_ expose the position (because that kinda defeats the point). But this is also extremely useful and looks cleaner than consistently calling `IncrementInBounds` over and over again. Any ideas on how to get the position a different way? Or is this good enough?

Reply:
lol, i wrote a comment above (in the cpp file, for this function) saying "danger! danger! danger!" before I saw this.

Reply:
Depends on why you need the position, really. I'll circle back after I read the rest of the PR.